apps/docs/.vitepress/config.ts

```diff
@@ -81,6 +81,10 @@ export default defineConfig({
             text: 'LQIP',
             link: '/lqip',
           },
+          {
+            text: 'Configure',
```

Comment:
Nit pick, but I would prefer for this page to have a title (and headline) of "Configuration".